config/routes.rb

```diff
@@ -6,9 +6,14 @@
   get "up" => "rails/health#show", as: :rails_health_check
 
   # Defines the root path route ("/")
-  # root "posts#index"
+  root "users#index"
   
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:new, :create]
+      member do
+        post 'like'
+      end
```

Comment:
- [ ] The like route should point to the Likes controller, just like the comments routes.

  ```
  resources :likes, only: [:create]
  ```